broker/src/main/java/io/moquette/broker/unsafequeues/PagedFilesAllocator.java

```diff
@@ -36,11 +46,30 @@ interface AllocationListener {
         this.segmentSize = segmentSize;
         this.lastPage = lastPage;
         this.lastSegmentAllocated = lastSegmentAllocated;
-        this.currentPage = openRWPageFile(this.pagesFolder, this.lastPage);
+        this.currentPage = getPageFile(this.lastPage);
+    }
+
+    private MappedByteBuffer getPageFile(int pageId) throws QueueException {
```

Comment:
I would rename, get<something> is confusing, a getter with parameter. Better something more explicit, I would propose 
```suggestion
    private MappedByteBuffer openOrRetrievePageFile(int pageId) throws QueueException {
```

Reply:
Good point about not making getters with parameters, or getters that do more complicated stuff than just getting...
Fixed it!

Reply:
Also added the changelog entry.